avocado/core/html.py

```diff
@@ -1,289 +0,0 @@
-
-        for top_dir in pkg_resources.resource_listdir(module, base_path):
-            rsrc_dir = base_path + '/%s' % top_dir
-            if pkg_resources.resource_isdir(module, rsrc_dir):
-                rsrc_files = pkg_resources.resource_listdir(module, rsrc_dir)
-                for rsrc_file in rsrc_files:
-                    source = pkg_resources.resource_filename(
-                        module,
-                        rsrc_dir + '/%s' % rsrc_file)
-                    dest = os.path.join(
-                        os.path.dirname(os.path.abspath(self.output)),
-                        top_dir,
-                        os.path.basename(source))
-                    pkg_resources.ensure_directory(dest)
-                    shutil.copy(source, dest)
-
-    def _render_report(self):
-        context = ReportModel(result=self,
-                              html_output=self.output)
-        template = pkg_resources.resource_string(
-            'avocado.core',
-            'resources/htmlresult/templates/report.mustache')
-
-        # pylint: disable=E0611
-        try:
-            if hasattr(pystache, 'Renderer'):
-                renderer = pystache.Renderer('utf-8', 'utf-8')
-                report_contents = renderer.render(template, context)
-            else:
-                from pystache import view
-                v = view.View(template, context)
-                report_contents = v.render('utf8')  # encodes into ascii
-                report_contents = codecs.decode("utf8")  # decode to unicode
-        except UnicodeDecodeError as details:
-            # FIXME: Removeme when UnicodeDecodeError problem is fixed
-            import logging
-            ui = logging.getLogger("avocado.app")
-            ui.critical("\n" + ("-" * 80))
-            ui.critical("HTML failed to render the template: %s\n\n",
-                        template)
-            ui.critical("-" * 80)
-            ui.critical("%s:\n\n", details)
-            ui.critical("%r\n\n", self.result_dict)
-            ui.critical("%r", getattr(details, "object", "object not found"))
-            ui.critical("-" * 80)
-            raise
-
-        self._copy_static_resources()
-        with codecs.open(self.output, 'w', 'utf-8') as report_file:
-            report_file.write(report_contents)
-
-        if self.args is not None:
-            if getattr(self.args, 'open_browser', False):
-                # if possible, put browser in separate process group, so
-                # keyboard interrupts don't affect browser as well as Python
-                setsid = getattr(os, 'setsid', None)
-                if not setsid:
-                    setsid = getattr(os, 'setpgrp', None)
-                inout = file(os.devnull, "r+")
-                cmd = ['xdg-open', self.output]
-                subprocess.Popen(cmd, close_fds=True, stdin=inout, stdout=inout,
-                                 stderr=inout, preexec_fn=setsid)
```

Comment:
Oh, you didn't, because it merges into the existing file... ignore my comment...